lasagne/layers/dnn.py

```diff
@@ -1,6 +1,4 @@
 import theano
-from theano.sandbox.cuda import dnn
-
```

Comment:
The empty line was on purpose; PEP 8 suggests to group imports into standard library, third-party, and internal (https://www.python.org/dev/peps/pep-0008/#imports).